### ISSUE TITLE
RTM abnormal close 1006 - Messages not sent

### ISSUE THREAD

**Issue by rickul** [howdyai/botkit#463]
Hi!
We have a problem with the RTM. When starting some conversations, the RTM connections is (always) lost with a 1006 abnormal close. This only happens in one specific channel, in others this never happens.
When it happens, RTM is reconnected automatically, but the messages are not being sent. I think, they should be sent after a reconnect. A comment in bot.send() in Slackbot_worker.js says:
/**
- The RTM failed and for some reason it didn't get caught
- elsewhere. This happens sometimes when the rtm has closed but
- We are sending messages anyways.
- Bot probably needs to reconnect!
  */

**Comment by rickul** [comment]
**Feature/Workaround Request**
Due to this bug we cannot trust in the RTM. But botkit conversations use both, RTM if possible and API calls if necessary.

Could you add something like a "forceAPICall" option to botkit to never use RTM:

```
var controller = Botkit.slackbot({
  ...
  forceAPICalls: true,
  ...
})
```

**Comment by peterswimm** [comment]
Issue solved by #474
